From a08f848d4cd4c8c6e055a1182542d053a0a32c6b Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@aspartame.nwnk.net>
Date: Sun, 18 Nov 2007 11:57:01 -0500
Subject: [PATCH] Bump DEFAULT_DPI to 96.

75 is just nonsense.
(cherry picked from commit db9ae863536fff80b5463d99e71dc47ae587980d)
---
 hw/xfree86/common/xf86Priv.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hw/xfree86/common/xf86Priv.h b/hw/xfree86/common/xf86Priv.h
index b6fbd3a..5a3cba3 100644
--- a/hw/xfree86/common/xf86Priv.h
+++ b/hw/xfree86/common/xf86Priv.h
@@ -119,7 +119,7 @@ extern RootWinPropPtr *xf86RegisteredPropertiesTable;
 #define DEFAULT_LOG_VERBOSE	3
 #endif
 #ifndef DEFAULT_DPI
-#define DEFAULT_DPI		75
+#define DEFAULT_DPI		96
 #endif
 
 #define DEFAULT_UNRESOLVED	TRUE
-- 
1.5.5.1

